.github/workflows/run_vi_analyzer.yml

```diff
@@ -7,14 +7,39 @@ on:
 jobs:
   run_vi_analyzer:
     name: Run VI Analyzer
-    runs-on: self-hosted
+    runs-on: 
+      - self-hosted
+      - windows
+      - x64
+      - rdss-measlinkbot-win-10-lv64
+    strategy:
+      matrix:
+        version: ["2021", "2022", "2023"]
+      fail-fast: false
+    timeout-minutes: 90
     steps:
+    - name: Enable git long paths
+      run: git config  --global core.longpaths true
+
     - uses: actions/checkout@v4
 
     - uses: actions/setup-python@v5
       with:
         python-version: '3.10'
 
+    - name: Generate port number
+      id: port_number
+      run: |
+        if (${{ matrix.version }} -eq "2020") {
```

Comment:
2020 isn't part of the matrix so you can delete this.

Reply:
With my latest comment about running only on one version, you can probably just delete most or all of this code.

Reply:
Deleted.